.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: python
 python:
-  - "3.5"
-  - "3.6"
+  - "3.7"
```

Comment:
Can we get around this by adding dataclasses as an import?  It would be good to support 3.6 since that's the default python3 on Ubuntu 18.
https://pypi.org/project/dataclasses/

Reply:
Using dataclasses makes sense - probably a better solution than insisting on python 3.7 (didn't realize that bmt requires 3.7 until the CI failed)